app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -299,7 +303,17 @@ class SampleView extends React.Component {
   renderSampleReportError = () => {
     const { pipelineRun, sample } = this.props;
     let status, message, linkText, issueType, link;
-    switch (pipelineRun.known_user_error) {
+    switch (
+      sample.upload_error || (pipelineRun && pipelineRun.known_user_error)
```

Comment:
Out of the scope of this PR, but I suggest we should add a ticket and refactor there error fields to more generic versions, like:
```
error_type = enum(UPLOAD_ERROR, KNOWN_USER_ERROR, SYSTEM_ERROR)
error_code = "SOME_ERROR_CODE"
error_description = "An extended error description"
error_additional_details = {...} # object with additional details
error_internal_details = {...} # object with internal details that can be used internally (ex: stacktrace, ids, service error messages, etc.)
```
```